<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>10</y>
      <width>630</width>
      <height>590</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="targetImageLabel">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>310</width>
        <height>310</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="resultImageLabel">
      <property name="geometry">
       <rect>
        <x>312</x>
        <y>1</y>
        <width>310</width>
        <height>310</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="loadTargetImageButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>420</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Target Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="colorCorrectionButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>420</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Color correction</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>620</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadMainImageButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>620</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Main Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainImageLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>720</width>
      <height>720</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLab_no_2">
    <property name="title">
     <string>lab no. 2</string>
    </property>
   </widget>
   <addaction name="menuLab_no_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1330</x>
     <y>671</y>
    </hint>
    <hint type="destinationlabel">
     <x>1370</x>
     <y>674</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
